features/poller/monitors/core/src/main/java/org/opennms/netmgt/poller/monitors/PageSequenceMonitor.java

```diff
@@ -355,6 +356,10 @@ public void process(HttpRequest request, HttpContext ctx) throws HttpException,
                     method.setQueryParameters(expandParms(svc));
                 }
 
+                for(final org.opennms.netmgt.config.pagesequence.Header header : m_page.getHeaders()) {
+                    method.setHeader(new BasicHeader(header.getName(), header.getValue()));
```

Comment:
Should probably also add some logging statements which header is used.
Also what happens if header name and value is empty?

Reply:
done